assets/src/scripts/components/graph/state/layout.js

```diff
@@ -130,13 +130,20 @@ export const getAxisYBBox = memoize(opts => createSelector(
  * @param  {Object} state   Redux state
  * @return {Object}         {x, y, width, height} of viewBox
  */
-export const getViewBox = memoize(opts => createSelector(
+export const getViewBox = memoize((opts) => createSelector(
     getContainerSize(opts),
     getAxisYBBox(opts),
     (containerSize, axisYBBox) => {
-        const aspectRatio = containerSize.height / containerSize.width || 0;
+        let aspectRatio;
+        if (opts.graphType === 'water-levels') {
+            aspectRatio = containerSize.height / containerSize.width || 0;
+        } else if (opts.graphType === 'construction') {
+            // add an arbitrary number to the aspect ratio to keep it long and not wide
+            aspectRatio = containerSize.height / containerSize.width + 1 || 0;
+        }
         const width = containerSize.width + axisYBBox.width + FOCUS_CIRCLE_RADIUS;
```

Comment:
I did take out the addition of the FOCUS_CIRCLE_RADIUS, and you seem to be right about it not being needed (I thought the FOCUS_CIRCLE_RADIUS addition was used to prevent some cropping of the focus circle). However, the change didn't seem to help with the correction factor.